.gitignore

```diff
@@ -1,2 +1,4 @@
 # ignore Mac OS temp file
 .DS_Store
+.idea/
```

Comment:
Can this be a separate pull? It will be tagged weirdly in the git history otherwise.